src/gui/guiFormSpecMenu.cpp

```diff
@@ -641,6 +641,12 @@ void GUIFormSpecMenu::parseCheckbox(parserData* data, const std::string &element
 	m_fields.push_back(spec);
 }
 
+void GUIFormSpecMenu::parseRealCoordinates(parserData* data, const std::string &element)
+{
+	auto description = std::get<std::string>(data->parser_type);
```

Comment:
`parser_type` is not the same as the previous `description`.

Reply:
i'll check this afternoon

Reply:
yep exact, it's element not description, i think we have a naming issue on those functions.